src/components/Text/Text.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import "./text.css";
+
+export interface TextProps {
+  /**
+   * Text variants
+   */
+  variant?:
+    | "header"
+    | "subheader"
+    | "body"
+    | "body-small"
+    | "subtext"
+    | "header-alt"
+    | "subheader-alt"
+    | "body-alt"
+    | "subtext-alt";
+  /**
+   * Text color to use (accepts hex, eg: #212121 or #FF0000)
+   */
+  color?: string;
+  /**
```

Comment:
Can add a fontSize field? Should be an optional field that can take in either "xs", "sm", "lg", "xl" etc. For now the font-size values can be some number which looks correct while waiting for figma